src/Microsoft.VisualBasic.Core/src/Microsoft.VisualBasic.Core.vbproj

```diff
@@ -108,8 +108,4 @@
     <Reference Include="System.Threading.Tasks" />
     <Reference Include="System.Threading.Thread" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetsUap)' == 'true'">
-    <!-- UAP is missing default imports -->
-    <Import Include="Microsoft.VisualBasic;System;System.Collections;System.Collections.Generic;System.Diagnostics;System.Linq;System.Xml.Linq;System.Threading.Tasks" />
```

Comment:
Why can we remove these imports now?

Reply:
Previously I had added them because UAP didn't get the defaults.  Now I just changed the source to specify them rather than rely on them being passed in.  See https://github.com/dotnet/sdk/blob/2eb6c546931b5bcb92cd3128b93932a980553ea1/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.VisualBasic.targets#L13-L39.